pkg/errors/codes.go

```diff
@@ -93,6 +93,23 @@ func ErrorDatabaseInsert(err error) (code int, body gin.H) {
 	return http.StatusBadRequest, body
 }
 
+//	{
+//		"type":   "/errors/database",
+//		"title":  "Failed to insert into database",
+//		"status": http.StatusBadRequest,
+//		"error":  err.Error(),
+//	}
+func ErrorDatabaseUpdate(err error) (code int, body gin.H) {
```

Comment:
I've copied this from the other example in this file, but It seems to me we should return a internal server error instead of a bad request on database access failure.